drivers/python/rethinkdb/ast.py

```diff
@@ -1483,7 +1486,7 @@ class Sync(RqlMethodQuery):
     st = 'sync'
 
 
-class Branch(RqlTopLevelQuery):
+class Branch(RqlMethodQuery):
```

Comment:
Ok, likewise in the JS driver should I revert [this line](https://github.com/rethinkdb/rethinkdb/pull/4442/files#diff-029874a8dbb2a1f773b7980319d494b5R978)?